inc/class-cli.php

```diff
@@ -128,16 +115,36 @@ function( $post ) use ( &$errors, &$confirmed, &$queue ) {
 				'errors'    => $batch_errors,
 			] = $this->run_audit_queue( $queue );
 
-			// Reset the queue.
-			$queue = [];
-
 			$confirmed += $batch_confirmed;
 
 			foreach ( $batch_errors as $post_id => $reason ) {
 				$errors[] = compact( 'post_id', 'reason' );
 			}
 		}
 
+		return compact( 'confirmed', 'errors' );
+	}
+
+	/**
+	 * Audit the site against Jetpack ES.
+	 *
+	 * ## OPTIONS
+	 *
+	 * [--fix]
+	 * : If present, script will attempt to fix any posts that fail the audit.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     wp jetpack sync-ep audit_posts
+	 */
+	public function audit_posts( $args, $assoc_args ) {
+		$fix = ! empty( $assoc_args['fix'] );
```

Comment:
Something I picked up in my travels that I really like is prefixing booleans with `should` or `is`, i.e. `$should_fix`

Reply:
I like that! thanks for sharing